### ISSUE TITLE
cannot use hook.Entries (type []logrus.Entry) as type []*logrus.Entry in return argument

### ISSUE THREAD

**Issue by frankegoesdown** [stellar/go#565]
after import  package `"github.com/stellar/go/clients/horizon"`

compiler says
```
go/src/github.com/stellar/go/support/log/entry.go:125:14: cannot use hook.Entries (type []logrus.Entry) as type []*logrus.Entry in return argument
```

**Comment by bartekn** [comment]
Very likely that you did not use `glide` to install dependencies: https://github.com/stellar/go#dependencies

**Comment by bartekn** [comment]
Sorry I read `services/horizon` instead of `clients/horizon`.

**Comment by frankegoesdown** [comment]
@bartekn I hotfix it in my local repo with change `[]*logrus.Entry` to `[]logrus.Entry` in 2 files

**Comment by bartekn** [comment]
What dependency management tool are you using in your workflow?

**Comment by frankegoesdown** [comment]
Just run example build_transaction
https://github.com/stellar/go/tree/master/examples/build_transaction

**Comment by italonascimento** [comment]
I'm getting the same error here, using dep to manage dependencies. Any idea what is it about?

**Comment by bartekn** [comment]
`glide` seems to ignore transitive dependencies. This is one of many reasons we switched to `dep` recently. You have two options:
1. Switch to `dep` too, and update Git revision of `github.com/stellar/go` to 435de795b0e1ae4ea4638344c5637bff14e31c99 or later - this is where we started using `dep` (recommended).
2. If you really want to stick to `glide` add `github.com/sirupsen/logrus` at `68cec9f21fbf3ea8d8f98c044bc6ce05f17b267a` to your glide.yaml and/or glide.lock files.
